# This msg file defines the path for rearranging an object
# which is consisted of three trajectories
# (1) transit trajectory (pre-picking + picking)
# (2) transfer trajectory (post-picking + pre-placing + placing)
# (3) finish trajectory (post-placing)

uniform_object_rearrangement/ArmTrajectory transit_trajectory
uniform_object_rearrangement/ArmTrajectory transfer_trajectory
uniform_object_rearrangement/ArmTrajectory finish_trajectory